Documentation/install/system_requirements.rst

```diff
@@ -104,6 +104,41 @@ linked, either choice is valid.
    Users running Linux 4.10 or earlier with Cilium CIDR policies may face
    :ref:`cidr_limitations`.
 
+L7 proxy redirection currently uses ``TPROXY`` iptables actions as well
+as ``socket`` matches. For L7 redirection to work as intended kernel
+configuration must include the following modules:
+
+.. code:: bash
+
+        CONFIG_NETFILTER_XT_TARGET_TPROXY=m
+        CONFIG_NETFILTER_XT_MATCH_MARK=m
+        CONFIG_NETFILTER_XT_MATCH_SOCKET=m
+
+.. note::
+
+   As noted above, ``xt_socket`` kernel module is generally needed for
```

Comment:
Given that this is in the same section, I don't think that this initial introductory sentence adds any additional information.

Reply:
deleted